src/Jobs/Recruit.Vacancies.Jobs/DashboardGenerator/DashboardCreator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Esfa.Recruit.Vacancies.Client.Infrastructure.Services;
+using System.Threading.Tasks;
+
+namespace Esfa.Recruit.Vacancies.Jobs.DashboardGenerator
+{
+    public class DashboardCreator
```

Comment:
Do we need this class? Should/can we just inject the IDashboardService straight into DashboardGeneratorJob?